Heffner_Daniel/Test_Driven_Development/registration_rspec/spec/models/user_spec.rb

```diff
@@ -0,0 +1,33 @@
+require 'spec_helper'
+
+RSpec.describe "User Model", type: :model do
+	
+	before do
+		@testy = User.create(first_name:'Testy', last_name:'McTesterson', email:'testy@tester.son', password:'password', password_confirmation:'password');
+	end
+
+	it 'requires a valid email' do
```

Comment:
I like making variables for things I am asserting on, so creating an "invalid_email" variable and then using it. It creates easily readable tests. 

You can also make a method right in the file called create_new_user and set the variables you are testing.